relay/http.go

```diff
@@ -21,9 +21,10 @@ import (
 
 // HTTP is a relay for HTTP influxdb writes
 type HTTP struct {
-	addr   string
-	name   string
-	schema string
+	addr     string
+	name     string
+	schema   string
+	response int
```

Comment:
I would rather see a name like `pingResponseCode` and have it separated by a newline so that it's visually separate from the above group of related fields.